main.go

```diff
@@ -1433,34 +1432,32 @@ func (c MainController) ListFBEventsHandler(w http.ResponseWriter, r *http.Reque
 	// start & end time (optional)
 	u, _ := url.Parse(r.URL.String())
 	params := u.Query()
-	var startTimeStr string
-	var endTimeStr string
-	stringFormat := "^[0-9]{4}-[0-9]{2}-[0-9]{2}$|^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}$"
-	if startTime, ok := params["start_time"]; ok {
-		startTimeStr = startTime[0]
-		match, _ := regexp.MatchString(stringFormat, startTimeStr)
-		if !match {
-			writeJSON(w, map[string]interface{}{
+
+	var startTime, endTime time.Time
+	var err error
+
+	if startTimeStr, ok := params["start_time"]; ok {
+		startTime, err = time.Parse(time.RFC3339, startTimeStr[0])
+		if err != nil {
+			writeJSON(w, map[string]string{
 				"error": "start_time format incorrect",
 			})
-			return
```

Comment:
Why remove the returns after writeJSON?

Reply:
That was a mistake. Thanks for catching.